utils/text2token.py

```diff
@@ -54,7 +54,7 @@ def get_parser():
         "-t",
         type=str,
         default="char",
-        choices=["char", "phn"],
+        choices=["char", "phn", "wrd"],
```

Comment:
ESPNet2 does not use functions in `utils` anymore. Meanwhile, I did not find where it is used in preparation. Could you remove this it not used?